packages/common/src/types/offers.ts

```diff
@@ -73,3 +73,17 @@ export type PremintParametersStruct = {
   reservedRangeLength: BigNumberish;
   to: string;
 };
+
+export enum Side {
+  Ask = 0,
+  Bid = 1,
+  Wrapper = 2
```

Comment:
if possible, I'd not use 0
```suggestion
  Ask = 1,
  Bid = 2,
  Wrapper = 3
```

Reply:
These values are the ones used in smart contract.
Creating a shift in coreSDK compared to the smart contract would raise confusion...